app/code/community/Bolt/Boltpay/etc/config.xml

```diff
@@ -248,41 +248,41 @@
     <payment>
       <boltpay>
         <active>1</active>
+        <test>0</test>
         <hide_on_checkout>0</hide_on_checkout>
-        <model>boltpay/payment</model>
+        <add_button_everywhere>0</add_button_everywhere>
+      </boltpay>
+      <additional_options>
+        <enable_product_page_checkout>0</enable_product_page_checkout>
+        <product_page_checkout_selector>.btn-cart</product_page_checkout_selector>
+        <additional_css>
```

Comment:
If you mean the css that follows <additional_css> then I'd prefer to leave it as is so it looks better on the actual page. Otherwise there'd be some extra tabs at the beginning on the actual admin page.